src/components/Answers/__tests__/Answers-test.tsx

```diff
@@ -0,0 +1,92 @@
+/** @jsx h */
+
+import { h } from 'preact';
+import { render } from '@testing-library/preact';
+import Answers, { AnswersProps } from '../Answers';
+
+const defaultProps: AnswersProps = {
+  hits: [],
+  isLoading: false,
+  cssClasses: {
+    root: 'root',
+    header: 'header',
+    emptyRoot: 'empty',
+    loader: 'loader',
+    list: 'list',
+    item: 'item',
+  },
+  templateProps: {
+    templates: {
+      header: 'header',
+      loader: 'loader',
+      item: 'item',
+    },
+  },
+};
+
+describe('Answers', () => {
+  describe('Rendering', () => {
+    it('renders without anything', () => {
+      const { container } = render(<Answers {...defaultProps} />);
+      expect(container.querySelector('.root')!.classList).toContain('empty');
```

Comment:
We can use `toHaveClass`.

Reply:
thanks for the suggestion https://github.com/algolia/instantsearch.js/pull/4581/commits/4f68c9e5503356e88d05edc1da21e19ccde27411 for all the three comments you recently left.